legend-engine-xt-relationalStore-bigquery-execution-tests/src/test/java/org/finos/legend/engine/plan/execution/stores/relational/connection/test/ExternalIntegration_TestConnectionAcquisitionWithFlowProvider_BigQuery.java

```diff
@@ -43,7 +43,7 @@
 import static org.junit.Assert.assertTrue;
 import static org.junit.Assert.fail;
 
-public class ExternalIntegration_TestConnectionAcquisitionWithFlowProvider_BigQuery extends DbSpecificTests
+public class ExternalIntegration_TestConnectionAcquisitionWithFlowProvider_BigQuery extends RelationalConnectionTest
```

Comment:
Remove the connection details from code and instead load it from a json file containing the connection. Also delete all the big query tests outside this module. If you search for bigquery across the legend-engine project, only modules containing bigquery in their name, should have bigquery code.

Reply:
Done.

Note: There are tests in authentication module for Auth strategies supported with BigQuery. It doesn't make sense to bring them into BigQuery module as they don't interact with the database itself.